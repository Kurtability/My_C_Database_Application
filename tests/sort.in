sort a
set a 3 2 1 6 5 4
sort a
get a
append a 1 1 1 2 2 2
get a
push a 2 1
get a
sort a
get a
bye
