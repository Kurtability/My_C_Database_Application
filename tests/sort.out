> sort a 
no such key

> set a 3 2 1 6 5 4
ok

> sort a
ok

> get a
[1 2 3 4 5 6]

> append a 1 1 1 2 2 2
ok

> get a 
[1 2 3 4 5 6 1 1 1 2 2 2]

> push a 2 1
ok

> get a 
[1 2 1 2 3 4 5 6 1 1 1 2 2 2]

> sort a
ok

> get a
[1 1 1 1 1 2 2 2 2 2 3 4 5 6]

> bye
bye
